README.rst

```diff
@@ -2,6 +2,76 @@
 chdkptp.py
 ==========
 
+This repo is a fork from  `chdkptp.py <https://github.com/jbaiter/chdkptp.py>`_
```

Comment:
Could you fix the repository URLs?

Reply:
Sure, but which should be the correct one?